### ISSUE TITLE
Specify template as "content' in JS API

### ISSUE THREAD

**Issue by meggers** [sir-dunxalot/ember-flash-messages#19]
It would be nice to be able to specify a partial template to be used as the content as well as an HTML string, as an alternative to using the standalone component.

**Comment by sir-dunxalot** [comment]
That's a really good idea! It might make more sense to allow a `partial` (or `template`) option in place of a `content` option to avoid confusion.

I believe Ember is trying to move away from partials to always use components but we can make it forwards compatible pretty easily.

Happy to accept a PR for this. Otherwise, I'll add it to my backlog.

**Comment by meggers** [comment]
Agreed, a different option is a good idea. I'll start looking into it!

**Comment by sir-dunxalot** [comment]
Ok, sounds great. I'll keep this thread updated if I find some spare time so we don't double up on the work!
